stake-pool/js/src/utils/program-address.ts

```diff
@@ -28,9 +28,14 @@ export async function findStakeProgramAddress(
   programId: PublicKey,
   voteAccountAddress: PublicKey,
   stakePoolAddress: PublicKey,
+  seed?: number,
 ) {
   const [publicKey] = await PublicKey.findProgramAddress(
-    [voteAccountAddress.toBuffer(), stakePoolAddress.toBuffer()],
+    [
+      voteAccountAddress.toBuffer(),
+      stakePoolAddress.toBuffer(),
+      seed ? new Uint8Array([seed]) : Buffer.alloc(0),
```

Comment:
This won't give us four LE bytes for a `u32`. I think what you had originally was better, or `u32.encode()` from `buffer-layout`.

Reply:
Or even better: `buffer.writeUInt32LE()`.

You could do something like:

```ts
      seed ? Buffer.alloc(4).writeUint32LE(seed) : Buffer.alloc(0),
```

Reply:
Ahh sorry btw. I had submitted my fix before I saw your comment. Let me know if you prefer that

Reply:
`Buffer.alloc(4).writeUint32LE(seed)` returns `number` so it would need to be converted if you prefer that.

Reply:
I'm good with `toArrayLike`!